From 58030d04e2bb8c14d36408f9695d4ec02ff91460 Mon Sep 17 00:00:00 2001
From: Ankit Jain <ankit.jain@microsoft.com>
Date: Wed, 21 Aug 2019 12:55:27 -0400
Subject: [PATCH] [mono] Remove Microsoft.DotNet.MSBuildSdkResolver on linux
 for the

.. bootstrap build. This is because it depends on a native lib,
libhostfxr*, which is not present in the bootrap msbuild, which breaks
the resolver and thus the build. Removing this should be fine, because
msbuild then just falls back to the bundled SDKs.
---
 eng/cibuild_bootstrapped_msbuild.sh | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/eng/cibuild_bootstrapped_msbuild.sh b/eng/cibuild_bootstrapped_msbuild.sh
index b6da0a9c32..bc4d58a92f 100755
--- a/eng/cibuild_bootstrapped_msbuild.sh
+++ b/eng/cibuild_bootstrapped_msbuild.sh
@@ -57,6 +57,11 @@ function DownloadMSBuildForMono {
     # rename just to make it obvious when reading logs!
     mv $artifacts_dir/msbuild $mono_msbuild_dir
     chmod +x $artifacts_dir/mono-msbuild/MSBuild.dll
+
+    if [[ `uname -s` != 'Darwin' ]]; then
+        # with no .so available, this ends up breaking the build
+        rm -Rf $mono_msbuild_dir/SdkResolvers/Microsoft.DotNet.MSBuildSdkResolver
+    fi
     rm "$msbuild_zip"
   fi
 }
 
